### DISCUSSION TITLE
Welcome to RedMarket Discussions!

### DISCUSSION THREAD

**Discussion by themubalion** [prokximus/RedMarket#16] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by themubalion** [comment]
Here we can discuss now related to the project

**Reply by SyedAdeebWebDesigning** [reply]
What's with commenting the code in Infinite scroll. Should we keep it commented or not

**Reply by themubalion** [reply]
Not needed now it was commented to save the code if anything goes wrong, also remove any unnecessary console logs I think I have done it but check it still

**Comment by themubalion** [comment]
changed the unnatural behaviour of the product image in slug.jsx

**Comment by SyedAdeebWebDesigning** [comment]
Just added login page and improved navbar as some links doesn't work properly on the mobile view.
`console.log('I am done for now')`

**Reply by themubalion** [reply]
> Just added login page and improved navbar as some links doesn't work properly on the mobile view. `console.log('I am done for now')`

Should've give the prompt instead logging the console
